src/app/app.component.css

```diff
@@ -0,0 +1,25 @@
+.inCentre{
```

Comment:
use some naming conventions on your style naming)
P.S recommending BEM

Reply:
Thanks for the advice.I rename on navbar-nav_position